### ISSUE TITLE
Direct tunnel (no proxy)

### ISSUE THREAD

**Issue by colemar** [proxytunnel/proxytunnel#64]
Is there any way to make a direct connection to a remote stunnel service?

This syntax does not work with proxytunnel 1.10.20210128 (Windows 10 x64):
```
>proxytunnel -v -d 111.122.133.144:443 -e
SSL enabled
error: failed to connect to local proxy: [125] Cannot assign requested address
```

If not possible, coould you consider this as a new feature?

**Comment by colemar** [comment]
I believe it makes sense and sometimes is useful to set up a TLS tunnel even without a proxy.

**Comment by 68420948** [comment]
Did you try stunnel in client mode on your local end? This should do what you are looking for, no need for proxytunnel at all.

**Comment by colemar** [comment]
> Did you try stunnel in client mode on your local end? This should do what you are looking for, no need for proxytunnel at all.

Yes, if I remember well it works, but stunnel client has its own quirks.  For example it cannot work as "Local" proxy with Putty, therefore I have to connect to an address like localhost:9999 (where stunnel listens) instead of remotehost:443 and set up many stunnels (one for each remote host).

**Comment by ftasnetamot** [comment]
Why proxytunnel?
Use socat
put a _**config**_ file in your _**.ssh**_ directory, with the following content:
```
host DESTINATION-HOST
    ProxyCommand socat - OPENSSL:%h:%p
```

**Comment by 68420948** [comment]
Excellent suggestion, @ftasnetamot!
